solutions/python_webserver/Makefile

```diff
@@ -18,7 +20,7 @@ rootfs:
 run:
 	test -f server.pid && kill -9 `cat server.pid` || true
 	$(MYST_EXEC) $(OPTS) rootfs /usr/local/bin/python3 /app/hello_server.py & echo $$! > server.pid
-	sleep 10
+	sleep 20
```

Comment:
Maybe some performance regressions. It used to launch within 10 seconds.

Reply:
I increased this because it fails on my machine, which is slower. It takes about 17 seconds to launch on my machine.